crucible-mir/src/Mir/Trans.hs

```diff
@@ -359,6 +391,26 @@ staticPlace did = do
             MirPlace (G.globalType gv) <$> globalMirRef gv <*> pure NoMeta
         Nothing -> mirFail $ "cannot find static variable " ++ fmt did
 
+-- variant of staticPlace for slices
+-- tpr is the element type; len is the length
+staticPlaces :: HasCallStack => C.TypeRepr tp -> Int -> M.DefId -> MirGenerator h s ret (MirPlace s)
+staticPlaces tpr len did = do
+    sm <- use $ cs.staticMap
+    case Map.lookup did sm of
+        Just (StaticVar gv) -> do
+            let tpr_found = G.globalType gv
+            Refl <- testEqualityOrFail (MirVectorRepr tpr) tpr_found $
+                "staticPlaces: wrong type: expected vector of " ++
+                show tpr ++ ", found " ++ show tpr_found
+            ref <- globalMirRef gv
+            ref' <- subindexRef tpr ref (R.App $ usizeLit 0)
+            -- XXX we seem to need to cons up a Crucible Atom (see
+            -- crucible/src/Lang/Crucible/CFG/Reg.hs) from len to put in here.
+            --let len' = error "oops"
+            let len' = R.App $ usizeLit $ fromIntegral len
+            return $ MirPlace tpr ref' (SliceMeta len')
```

Comment:
I don't think these comments are relevant anymore, since there is now a suitable `Atom` to use for the `SliceMeta` length. Delete these comments?

Reply:
They are not. Removed in 3444637b455edf15c14963403aa69cd2f4192f5e